interface/forms/button_ros/new.php

```diff
@@ -0,0 +1,667 @@
+			<input id="GI6" name="GI6" type="checkbox" value="on" >
+			<label for="GI6"> <?php echo xlt('Vomiting');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI7" name="GI7" type="checkbox" value="on" >
+			<label for="GI7"> <?php echo xlt('Hematemesis');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI8" name="GI8" type="checkbox" value="on" >
+			<label for="GI8"> <?php echo xlt('Abdominal Pain');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI9" name="GI9" type="checkbox" value="on" >
+			<label for="GI9"> <?php echo xlt('Jaundice');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI10" name="GI10" type="checkbox" value="on" >
+			<label for="GI10"> <?php echo xlt('Diarrhea');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI11" name="GI11" type="checkbox" value="on" >
+			<label for="GI11"> <?php echo xlt('Constipation');?></label>
+		</span>
+		</TD>	
+	
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="GI_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>	
+
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('GU:');?></B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+		<span class="checkbox-button">
+			<input id="GU1" name="GU1" type="checkbox" value="on" >
+			<label for="GU1"> <?php echo xlt('Hematura');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU2" name="GU2" type="checkbox" value="on" >
+			<label for="GU2"> <?php echo xlt('Change in Frequency of Urination');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU3" name="GU3" type="checkbox" value="on" >
+			<label for="GU3"> <?php echo xlt('Urinary Urgency');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU4" name="GU4" type="checkbox" value="on" >
+			<label for="GU4"> <?php echo xlt('Incontinence');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU5" name="GU5" type="checkbox" value="on" >
+			<label for="GU5"> <?php echo xlt('Slow Urination');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU6" name="GU6" type="checkbox" value="on" >
+			<label for="GU6"> <?php echo xlt('Irregular Periods');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU7" name="GU7" type="checkbox" value="on" >
+			<label for="GU7"> <?php echo xlt('Hot Flashes');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU8" name="GU8" type="checkbox" value="on" >
+			<label for="GU8"> <?php echo xlt('Post Menopausal Bleeding');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="GU_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>
+
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('MSK:');?></B>
```

Comment:
I always got shot down for puncuation in the information to be translated it breaks the translation